schema.sql

```diff
@@ -63,3 +63,28 @@ ALTER COLUMN id SET DEFAULT nextval('animals_id_seq');
 -- Finally, set the 'id' column as the primary key
 ALTER TABLE animals
 ADD PRIMARY KEY (id);
+
+-- Create the vets table
+CREATE TABLE vets (
+    id serial PRIMARY KEY,
+    name varchar(255),
+    age integer,
+    date_of_graduation date
+);
+
+-- Create the specializations table
+CREATE TABLE specializations (
+    vet_id integer,
+    species varchar(255),
+    FOREIGN KEY (vet_id) REFERENCES vets(id)
+);
+
+-- Create the visits table
+CREATE TABLE visits (
+    animal varchar(255),
+    vet_id integer,
+    visit_date date,
+    FOREIGN KEY (vet_id) REFERENCES vets(id)
+);
```

Comment:
- [ ] 👍 Great effort on restructuring the database tables! However, to ensure that you have a proper many-to-many relationship between the `animals` and `vets` tables, you should establish a connection between these two tables as well, as per the project requirement. I recommend following the guidance provided in the [Many to Many relationship documentation](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/lessons/relationships.md#many-to-many) to make these necessary updates.

- [ ] Since you are updating the table structure, it's important to thoroughly test your queries to verify that the outputs align with the new structure. If needed, update the screenshots in the PR Description to reflect the current state of the database.

By making these adjustments, you will ensure that your database structure complies with the project requirements and functions correctly. Keep up the good work! 👍🗃️📊